addons/payment/models/payment_acquirer.py

```diff
@@ -185,13 +185,22 @@ def _get_feature_support(self):
     @api.model
     def create(self, vals):
         image_resize_images(vals)
+        vals = self._check_journal_id(vals)
         return super(PaymentAcquirer, self).create(vals)
 
     @api.multi
     def write(self, vals):
         image_resize_images(vals)
+        vals = self._check_journal_id(vals)
         return super(PaymentAcquirer, self).write(vals)
 
+    def _check_journal_id(self, vals):
```

Comment:
Rebase issue I suppose (to move into its own commit).

Reply:
Hmm, what about the fact that it makes no sense for write(), as we're now resetting the journal every time you edit an acquirer?
Also:
`vals['journal_id'] = default_journal.id`
;-)